onnxruntime/contrib_ops/cuda/moe/ft_moe/moe_gemm_kernels_template.h

```diff
@@ -387,52 +404,104 @@ void MoeGemmRunner<T, WeightType>::dispatch_to_arch<EpilogueTag>(const T* A, con
 
 template <typename T, typename WeightType>
 template <typename EpilogueTag>
-void MoeGemmRunner<T, WeightType>::run_gemm<EpilogueTag>(const T* A, const WeightType* B, const T* weight_scales,
-                                                         const T* biases, T* C, int64_t* total_rows_before_expert,
-                                                         int64_t total_rows, int64_t gemm_n, int64_t gemm_k,
-                                                         int num_experts, cudaStream_t stream) {
+void MoeGemmRunner<T, WeightType>::profile_gemm<EpilogueTag>(const T* A, const WeightType* B, const T* weight_scales,
+                                                             const T* biases, T* C, int64_t* total_rows_before_expert,
+                                                             int64_t total_rows, int64_t gemm_n, int64_t gemm_k,
+                                                             int num_experts, cudaStream_t stream,
+                                                             MoEGemmConfigMap::type& best_config_map,
+                                                             int64_t key) {
   static constexpr bool is_weight_only = !std::is_same<T, WeightType>::value;
   static constexpr bool only_simt_configs = std::is_same<T, float>::value;
+
   std::vector<CutlassGemmConfig> candidate_configs = get_candidate_configs(sm_, is_weight_only, only_simt_configs);
   std::vector<int> occupancies(candidate_configs.size());
 
+  constexpr int warmup = 3;
+  constexpr int runs = 5;
```

Comment:
Is 5 enough?

Reply:
actually the results are pretty stable with 5. Changed to 10 since TRT uses 10